provider/aws/formation/app.json.tmpl

```diff
@@ -137,6 +148,50 @@
   {{ end }}
 {{ end }}
 
+{{ define "caches" }}
+  {{ with .Manifest.Caches }}
+  "CacheSubnetGroup" : {
```

Comment:
Seems like multiple caches should just be one redis server and different database on that server?